Example/AppDelegate+CarPlay.swift

```diff
@@ -56,7 +61,12 @@ extension AppDelegate: CarPlayManagerDelegate {
                                          detailText: FavoritesList.POI.timesSquare.subTitle)
         mapboxSFItem.userInfo = [CarPlayManager.CarPlayWaypointKey: Waypoint(location: FavoritesList.POI.mapboxSF.location)]
         timesSquareItem.userInfo = [CarPlayManager.CarPlayWaypointKey: Waypoint(location: FavoritesList.POI.timesSquare.location)]
-        let listSection = CPListSection(items: [mapboxSFItem, timesSquareItem])
+        
+        let multilegRouteItem = CPListItem(text: "Multileg route", detailText: "Liechtenstein")
+        let multilegRoute = Fixture.route(from: "multileg-route")
+        multilegRouteItem.userInfo = [CarPlayManager.CarPlayWaypointKey: multilegRoute.routeOptions.waypoints]
```

Comment:
Instead of bringing in the entire route as a fixture (which incurs a problem with importing TestHelper), could we just inline the waypoints?